### ISSUE TITLE
sklearnDev CI tests failing on sklearn.preprocessing.label module

### ISSUE THREAD

**Issue by stsievert** [dask/dask-ml#560]
Here's a traceback from #528 

``` python
______________ ERROR collecting tests/preprocessing/test_label.py ______________
tests/preprocessing/test_label.py:10: in <module>
    import dask_ml.preprocessing as dpp
dask_ml/preprocessing/__init__.py:3: in <module>
    from ._encoders import OneHotEncoder
dask_ml/preprocessing/_encoders.py:10: in <module>
    from .label import _encode, _encode_dask_array
dask_ml/preprocessing/label.py:10: in <module>
    from sklearn.preprocessing import label as sklabel
/usr/share/miniconda/envs/dask-ml-t…
/usr/share/miniconda/envs/dask-ml-test/lib/python3.7/site-packages/sklearn/utils/deprecation.py:143: in _raise_dep_warning_if_not_pytest
    warnings.warn(message, DeprecationWarning)
E   DeprecationWarning: The sklearn.preprocessing.label module is  deprecated in version 0.22
 and will be removed in version 0.24. The corresponding classes / functions should instead be
 imported from sklearn.preprocessing. Anything that cannot be imported from
 sklearn.preprocessing is now part of the private API.
```
